CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ without shipping useless bits. TBD!
 - Support for `react@0.14.x` has been dropped, although _our_ tests pass
 this is likely to break in other apps. We now require `react@^0.14.9`, or
 anything on the `15.x` branch
+- `<RequireForAccess/>` now only accepts a *single* child node
+
+### Removed
```

Comment:
Ah, I wanted to call it out as it _does_ change the output HTML in that we're removing a node. Maybe that is worth noting in the `Changes`? Maybe not?